BlueprintUI/Sources/Layout/LayoutSubelement.swift

```diff
@@ -1,8 +1,189 @@
+/// should also use the proxy to tell its corresponding element where to appear by calling the
+/// proxy’s ``place(at:anchor:size:)`` method. Do this once for each subview from your
+/// implementation of the layout’s ``CaffeinatedLayout/placeSubelements(in:subelements:cache:)``
+/// method.
+///
 public struct LayoutSubelement {
-    // TODO:
+
+    typealias SizeCache = HintingSizeCache
+
+    var identifier: ElementIdentifier
+    private var content: ElementContent
+    var measureContext: MeasureContext
+
+    var environment: Environment { measureContext.environment }
+    private var cache: HintingSizeCache { measureContext.node.sizeCache }
+
+    @Storage
+    private(set) var placement: Placement?
+
+    /// Optional attributes to apply to this subelement, such as opacity and transforms.
+    @Storage
+    public var attributes = Attributes()
+
+    // Once we are able to fully deprecate the old layout API we can remove the `LayoutType.Traits`
+    // and the `traits` parameter, and instead rely on setting `LayoutValueKey`s where needed.
+    private var traits: Any
+
+    init(
+        identifier: ElementIdentifier,
+        content: ElementContent,
+        measureContext: MeasureContext,
+        traits: Any
+    ) {
+        self.identifier = identifier
+        self.content = content
+        self.measureContext = measureContext
+        self.traits = traits
+    }
+
+    /// Assigns a position and size to a subelement.
+    ///
+    /// Call this method from your implementation of the `Layout` protocol’s
+    /// ``CaffeinatedLayout/placeSubelements(in:subelements:cache:)`` method for each subelement
+    /// arranged by the layout. Provide a position within the container’s bounds where the
+    /// subelement should appear, an anchor that indicates which part of the subelement appears at
+    /// that point, and a size.
+    ///
+    /// To learn the subelement's preferred size for a given proposal before calling this method,
+    /// you can call ``sizeThatFits(_:)`` method on the subelement.
+    ///
+    /// If you call this method more than once for a subelement, the last call takes precedence. If
+    /// you don’t call this method for a subelement, the subelement fills the bounds of its
+    /// container.
+    ///
+    /// - Parameters:
+    ///   - position: The place where the anchor of the subelement should appear in its container,
+    ///     relative to the container's bounds.
+    ///   - anchor: The unit point on the subelement that appears at `position`. You can use a
+    ///     built-in point, like ``UnitPoint/center``, or you can create a custom ``UnitPoint``.
+    ///   - size: The size of the subelement. In Blueprint, parents choose their children's size.
+    ///     You can determine a good size for a subelement by calling ``sizeThatFits(_:)`` on it.
+    public func place(
+        at position: CGPoint,
+        anchor: UnitPoint = .topLeading,
```

Comment:
Is `UnitPoint` used in the SwiftUI placement APIs?

Reply:
> Is `UnitPoint` used in the SwiftUI placement APIs?

[Yes](https://developer.apple.com/documentation/swiftui/layoutsubview/place(at:anchor:proposal:))